API/Parser/Parser.cs

```diff
@@ -83,6 +83,11 @@ public static class Parser
                 @"(?<Series>.*)(\b|_|)(S(?<Volume>\d+))",
                 RegexOptions.IgnoreCase | RegexOptions.Compiled,
             RegexTimeout),
+            // vol_001-1.cbz
```

Comment:
Put in your comment this is a format for MangaPy itself. So: // vol_001-1.cbz for MangaPy default naming convention